nixos/modules/services/development/jupyter/default.nix

```diff
@@ -37,14 +46,30 @@ in
       '';
     };
 
-    # NOTE: We don't use top-level jupyter because we don't
-    # want to pass in JUPYTER_PATH but use .environment instead,
-    # saving a rebuild.
-    package = lib.mkPackageOption pkgs [ "python3" "pkgs" "notebook" ] { };
```

Comment:
Restore the package option, but change it to jupyter. This allows overriding the package when needed.

Reply:
added the option to overwrite the jupyter package